content/software-security/what-makes-a-good-sbom.md

```diff
@@ -0,0 +1,41 @@
+
+## Basic SBOM Use Cases and Reqiured SBOM Data
+
+**Identify Vulnerable Components**: SBOMs can help organizations and individuals know the unfixed vulnerabilities in their software. By providing an inventory of components, this allows a software maintainer to check whether the versions of their component are associated with any known vulnerabilities. The software maintainer can then update or patch any components with known vulnerabilities.
+
+To enable this use case, SBOMs must contain information, at a minimum, about the component name and version. Because there are many different open source software package ecosystems, it is also advantageous to include information about the package ecosystem from which a component originates. This reduces the chances of a false positive when identifying potential vulnerabilities. A [package URL](https://github.com/package-url/purl-spec) (or purl) provides this ecosystem information.
+
+**Identifying Licenses**: SBOMs can also help organizations and individuals use open source software consistent with the licensing terms of all components. By identifying all components and associated licenses, software teams can understand the legal implications of any decision related to incorporating a particular component.
+
+To enable license analysis, an SBOM must contain information about the license or licenses associated with all components.
+
+Note: There are [other SBOM use cases](https://www.atlanticcouncil.org/in-depth-research-reports/issue-brief/the-cases-for-using-sboms/), such as mapping broader ecosystem risks, but this guide currently focuses on the arguably two most well-established SBOM use cases.
+
+## Measuring SBOM Quality
+
+The tools for measuring SBOM quality--like the overall concept of SBOM quality itself--are nascent.
+
+[SBOM Scorecard](https://github.com/eBay/sbom-scorecard) analyzes both major SBOM formats and returns a composite quality score indicating the extent to which an SBOM possesses key fields, including whether the components in the SBOM contain a purl and whether there are licenses associated with each component.
+
+[NTIA Conformance Checker](https://github.com/spdx/ntia-conformance-checker) analyzes whether [SPDX](https://spdx.dev/) SBOM documents posess the data fields associated with the so-called "NTIA minimum elements." [The U.S. National Telecommunications and Information Administration "minimum elements"](https://www.ntia.doc.gov/files/ntia/publications/sbom_minimum_elements_report.pdf) include the data fields deemed essential for basic SBOM use cases, including the identification of vulnerable components and the identification of component licenses.
+
```

Comment:
I think it would be valuable to include a little "learn more" section so people can find out more information. Maybe pointing to some of the policy work currently in flight?

Reply:
Added! Though I reference a couple chainguard blog posts, not policy work.